repository.json

```diff
@@ -793,6 +793,19 @@
 				}
 			]
 		},
+		{
+			"details":"https://github.com/PythonVoiceCodingPlugin/LSP-pyvoice",
+			"name": "LSP-pyvoice",
+			"author": "mpourmpoulis",
```

Comment:
please format properly with spaces. LSP-json can format for you

Reply:
reformaratted with LSP-json as requested! hope it is ok now!